interface/main/calendar/add_edit_event.php

```diff
@@ -185,6 +200,17 @@ function DOBandEncounter()
              }
      }
     }
+    // auto create encounter fot therapy group
+    if(!empty($_POST['form_gid'])){
+                                                                                // status Took Place is the check in of therapy group
+        if ($GLOBALS['auto_create_new_encounters'] && $event_date == date('Y-m-d') && $_POST['form_apptstatus'] == '=') {
```

Comment:
Gonna use below for my notes as I clarify the use of the above 3 lists:
1. Group Status list appears to be a subset of Appointment Status list and appears to be stored in calendar entry
2. Group Attendance Status list is separate and does not appear to be stored in calendar entry